src/test/java/com/kainos/ea/integration/RoleIntegrationTest.java

```diff
@@ -0,0 +1,29 @@
+package com.kainos.ea.integration;
+
+import com.kainos.ea.WebServiceApplication;
+import com.kainos.ea.WebServiceConfiguration;
+import com.kainos.ea.models.Role;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class RoleIntegrationTest {
+    private static final DropwizardAppExtension<WebServiceConfiguration> APP =
+            new DropwizardAppExtension<>(WebServiceApplication.class);
+
+    @Test
+    void getAllJobRoles_shouldReturnListOfJobRoles(){
```

Comment:
I would add a test for asserting an appropriate/graceful handling of any error(s) thrown when the app fails to get jobRoles.

Reply:
done